### PR TITLE
 added video and updated contributing section in documentation

### PR DESCRIPTION
**Pull request by jakepage91**

### CODE REVIEW

File: docs/Contributing/contribute.md [hunk 1]
```diff
@@ -5,17 +5,46 @@ sidebar_label: Contribute
...
+      left:0,
+      width:'100%',
+      height:'100%',
+    }}
+  >
+  </iframe>
+</div>
+
+---
+If you're in need of any assiatance at any stage of your contributing journey please don't hesitate to reach out to anybody in the `#general` or `#feedback` discord channels, also let us know if you want to be added to the private `#contributors` channel too. Or directly to `@mlabouardy` or `@JakePage` who will always be happy to help.
```
Comment (mlabouardy) [inline docs/Contributing/contribute.md:50]: assistance

File: docs/Contributing/contribute.md [hunk 2]
```diff
@@ -5,17 +5,46 @@ sidebar_label: Contribute
...
 - Afterwards, go ahead and **fork** the docs [repository](https://github.com/tailwarden/docs). Make any changes you want to your fork, and when you're ready to send those changes to us, go to your fork and create a new pull request.
 
 - If it takes longer than expected to get feedback from the `Tailwarden` team, head over to the [Discord Server](https://discord.tailwarden.com) and **ping** a Tailwarden staff member either in the `#general` or `#feedback` channel, unless you are a member of the private `contributor` channel. You can always request access to this channel. 
 
-- Once your pull request is created, a `Tailwarden` reviewer will provide clear and actionable feedback to you. As the owner of the pull request, it is your responsibility to modify it to address the feedback that has been provided.
+- Once your pull request has been opened it will be assigned to one or more reviewers. Those reviewers will do a thorough code review, looking for correctness, bugs, opportunities for improvement, documentation, comments, and style
```
Comment (mlabouardy) [inline docs/Contributing/contribute.md:17]: "has been opened, it will", maybe ?

**Review by mlabouardy — CHANGES_REQUESTED** [review]
A typo to be fixed

File: docs/Contributing/contribute.md (hunk 1 above)
Comment (jakepage91) [reply docs/Contributing/contribute.md]: Should be good now

File: docs/Contributing/contribute.md (hunk 2 above)
Comment (jakepage91) [reply docs/Contributing/contribute.md]: Done!

**Review by mlabouardy — APPROVED** [review]
Looks good, thanks

**Pull request merged**